package.json

```diff
@@ -69,7 +70,7 @@
     "dist-src": "cp -r ./src ./lib && find ./lib -name '__tests__' -type d -exec rm -r '{}' + && ./node_modules/.bin/babel lib --out-dir lib",
     "livereload": "NODE_ENV='development' ./node_modules/.bin/gulp docs:livereload",
     "npm-publish": "npm publish ./",
-    "test": "jest",
+    "test": "mv .babelrc .babelrc-tmp && jest && mv .babelrc-tmp .babelrc",
```

Comment:
1. We shouldn't be merging things in that don't pass tests. So that is a small worry for me.
2. This is only temporary. See [this](https://github.com/mesosphere/reactjs-components/commit/19662452dee1302fd6e742b6fc4799decc9a413e)